conda-requirements.txt

```diff
@@ -3,7 +3,7 @@ decorator
 msgpack-python>=0.5.2
 # Currently dask and numpy==1.16.0 clash
 numpy!=1.15.0,!=1.16.0
-pandas>=0.23.0, !=1.0.0 # https://github.com/pandas-dev/pandas/issues/31441
+pandas>=0.23.0, !=1.0.0 , <1.1 # https://github.com/pandas-dev/pandas/issues/31441
```

Comment:
@usha-nemani-jdas please also add this change to `requirements.txt§

Reply:
Yes done.